src/Workspaces/Remote/ServiceHub/Services/Host/RemoteHostService.cs

```diff
@@ -64,10 +64,10 @@ public RemoteHostService(Stream stream, IServiceProvider serviceProvider)
 
             if (TestData == null || !TestData.IsInProc)
             {
-                // Set this process's priority BelowNormal.
+                // Try setting this process's priority BelowNormal.
                 // this should let us to freely try to use all resources possible without worrying about affecting
                 // host's work such as responsiveness or build.
-                Process.GetCurrentProcess().PriorityClass = ProcessPriorityClass.BelowNormal;
+                Process.GetCurrentProcess().TrySetPriorityClass(ProcessPriorityClass.BelowNormal);
```

Comment:
💭 Where is `TrySetPriorityClass` defined?

Reply:
I forgot to commit the file! My bad